src/components/VideoCard/VideoCard.component.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './VideoCard.styles.css';
+
+function VideoCard({ data, id }) {
+  const renderHTML = (rawHTML) =>
+    React.createElement('div', { dangerouslySetInnerHTML: { __html: rawHTML } });
```

Comment:
Why are you using `dangerouslySetInnerHTML`? Using this could open you up to a cross-site scripting (XSS) attack
You need to make sure your HTML is structured properly and sanitized before inserting it into your page.

Reply:
I get some strange characters and only I investigate how to avoid that

one question 

Why my account was deactivated in the wizeline academy slack?